src/webgpu/util/math.ts

```diff
@@ -1641,6 +1641,84 @@ export function sparseMatrixF16Range(c: number, r: number): number[][][] {
   return kSparseMatrixF16Values[c][r];
 }
 
+/** Short list of f64 values of interest to test against */
+const kInterestingF64Values: number[] = [
+  kValue.f64.negative.min,
+  -10.0,
```

Comment:
nit:  I suggest having at least one normal values that is not integral.

E.g. 0.125  (a.k.a. 1/8)